spec/features/new_listing_spec.rb

```diff
@@ -2,6 +2,8 @@
   scenario "adding a new listing that is then visible" do
     add_listing1
     expect(page).to have_content 'testtitle'
+    expect(page).to have_content 'test description'
+    expect(page).to have_content '100'
     expect(page).not_to have_content 'sinatra'
```

Comment:
what is the reason for expecting the page not to have content sinatra? You could alternatively expect current_path if that is what you are trying to test with this line but I might be misunderstanding

Reply:
It's a practice to try and avoid the possibility of rspec throwing false positives when using 'expect to have_content' - because often sinatra can display an error page that contains the content you are looking for, which causes the test to pass, even though it's not working. This guards against that. I try and do this now after tearing my hair out for an hour trying to work out why I had a test passing when I could see quite clearly that it wasn't working haha - a coach gave me this practice to guard against it